code/modules/hydroponics/grown/banana.dm

```diff
@@ -137,11 +137,10 @@
 	filling_color = "#FFFF00"
 	bitesize = 5
 	
-/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/add_juice()
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/New()
```

Comment:
I have not tested this btw, so make sure you do! Compile it, run the code, throw it in the DA and see if you get bananium!